### PR TITLE
general cleanup

### PR DESCRIPTION
**Pull request by Nyameliaaaa**
this pr is an extension of #119

### CODE REVIEW

**Review by hkubus — APPROVED** [review]

File: src/commands/profile/view.ts [hunk 1]
```diff
@@ -93,7 +89,7 @@ export async function run(client: FluorineClient, interaction: CommandInteractio
     ctx.clip();
     ctx.drawImage(avatar, 30, 7, 110, 110);
 
-    const attachment = new MessageAttachment(canva.toBuffer(), 'profile.png');
+    const attachment = new MessageAttachment(canva.toBuffer(), 'profile?.png');
```
Comment (SuperchupuDev) [inline src/commands/profile/view.ts]:
```suggestion
    const attachment = new MessageAttachment(canva.toBuffer(), 'profile.png');
```
its literally the filename lol

**Review by SuperchupuDev — COMMENTED** [review]
also remove the polish `PROFILE_INVALID_USER` translation too

File: src/commands/profile/view.ts (hunk 1 above)
Comment (hkubus) [reply src/commands/profile/view.ts]: I AGREE

File: tsconfig.json [hunk 2]
```diff
@@ -9,15 +9,14 @@
...
+            "types/*": ["types/*"]
         },
         "allowSyntheticDefaultImports": true,
         "esModuleInterop": true,
         "skipLibCheck": true,
         "forceConsistentCasingInFileNames": true,
         "sourceMap": true
     }
-}
+}
```
Comment (SuperchupuDev) [inline tsconfig.json]:
```suggestion
}

```

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]

**Review by SuperchupuDev — APPROVED** [review]

**Comment by Nyameliaaaa** [comment]
> also remove the polish `PROFILE_INVALID_USER` translation too

will do when i fix the i18n strings

**Pull request merged**